src/Makefile.global.in

```diff
@@ -598,6 +598,13 @@ TAS         = @TAS@
 
 ##########################################################################
 #
+# Link to ZMQ
+
+LIBS := -lzmq $(LIBS)
```

Comment:
ZeroMQ is in C++ so I'm not sure how we can statically link it without compiling with `g++`.